examples/src/test/scala/io/snappydata/quickstart/ExampleTestSuite.scala

```diff
@@ -0,0 +1,109 @@
+      s"run '$snappyHome/quickstart/scripts/oltp_queries.sql';",
+      s"run '$snappyHome/quickstart/scripts/olap_queries.sql';",
+      s"run '$snappyHome/quickstart/scripts/olap_approx_queries.sql';",
+      "exit;"))
+
+
+    Job("io.snappydata.examples.AirlineDataJob", localLead, quickStartJar)
+
+    Job("io.snappydata.examples.CreateAndLoadAirlineDataJob", localLead, quickStartJar)
+
+    SparkSubmit("AirlineDataApp", appClass = "io.snappydata.examples.AirlineDataSparkApp", None,
+      confs = Seq("snappydata.store.locators=localhost:10334", "spark.ui.port=4041"),
+      appJar = quickStartJar)
+
+    SparkSubmit("PythonAirlineDataApp", appClass = "", None,
+      confs = Seq("snappydata.store.locators=localhost:10334", "spark.ui.port=4041"),
+      appJar = s"$snappyHome/quickstart/python/AirlineDataPythonApp.py")
+
+  }
+
+  test("QuickStart.scala script"){
+    SparkShell(Seq.empty[String], "--driver-memory=4g --driver-java-options=\"-XX:+UseConcMarkSweepGC\" \"-XX:+UseParNewGC\" \"-XX:+CMSClassUnloadingEnabled\" \"-XX:MaxNewSize=1g\" ",
+      scriptFile = s"$snappyHome/quickstart/scripts/Quickstart.scala")
+  }
+
+  test("JDBCExample") {
+    RunExample("JDBCExample", "snappydata.JDBCExample")
+  }
+
+  test("JDBCWithComplexTypes") {
+    RunExample("JDBCWithComplexTypes", "snappydata.JDBCWithComplexTypes")
+  }
+
+  test("CollocatedJoinExample") {
+    Job(s"$snappyExamples.CollocatedJoinExample",
+      localLead, quickStartJar)
+  }
+
+  test("CreateColumnTable") {
+    Job(s"$snappyExamples.CreateColumnTable",
+      localLead, quickStartJar, Seq(s"data_resource_folder=$snappyHome/quickstart/src/main/resources"))
+  }
+
+  test("CreatePartitionedRowTable") {
+    Job(s"$snappyExamples.CreatePartitionedRowTable",
+      localLead, quickStartJar)
+  }
+
+  test("CreateReplicatedRowTable") {
+    Job(s"$snappyExamples.CreateReplicatedRowTable",
+      localLead, quickStartJar)
+  }
+
+  test("WorkingWithObjects") {
+    Job(s"$snappyExamples.WorkingWithObjects",
+      localLead, quickStartJar)
+  }
+
+  test("WorkingWithJson") {
+    Job(s"$snappyExamples.WorkingWithJson",
+      localLead, quickStartJar,
+      Seq(s"json_resource_folder=$snappyHome/quickstart/src/main/resources"))
+  }
+
+  test("SmartConnectorExample") {
+    SnappyShell("smartConnectorSetup", Seq("connect client 'localhost:1527';",
+      s"CREATE TABLE SNAPPY_COL_TABLE(r1 Integer, r2 Integer) USING COLUMN;",
+      s"insert into SNAPPY_COL_TABLE VALUES(1,1);",
+      s"insert into SNAPPY_COL_TABLE VALUES(2,2);",
+      "exit;"))
+
+    RunExample("SmartConnectorExample", "snappydata.SmartConnectorExample")
+  }
+
+  test("StreamingExample") {
+    RunExample("StreamingExample", "snappydata.StreamingExample")
+
+  }
+
+  test("SynopsisDataExample") {
+    Job(s"$snappyExamples.SynopsisDataExample",
+      localLead, quickStartJar,
+      Seq(s"data_resource_folder=$snappyHome/quickstart/data"))
+  }
+
+}
```

Comment:
Python how-to example CreateTable.py needs to be added.

Since some of the examples can be run with run-example script and also as a job, may be we should run one or two of those those using both ways just to make sure run-example is not broken for those examples.